pushy/src/main/java/com/relayrides/pushy/apns/ApnsClientHandler.java

```diff
@@ -361,7 +361,7 @@ public void operationComplete(final ChannelPromise future) throws Exception {
     @Override
     public void userEventTriggered(final ChannelHandlerContext context, final Object event) throws Exception {
         if (event instanceof IdleStateEvent) {
-            assert PING_TIMEOUT < ApnsClient.PING_IDLE_TIME_MILLIS;
+            assert PING_TIMEOUT < this.apnsClient.idlePingIntervalMillis;
```

Comment:
For folks following this thread, we've discussed this in https://github.com/relayrides/pushy/issues/452#issuecomment-294855725.